src/tox_ansible/ansible/__init__.py

```diff
@@ -44,6 +46,41 @@ def is_ansible(self):
             path.join(self.directory, "galaxy.yml")
         )
 
+    @property
+    def is_global_molecule_config(self):
```

Comment:
In python functions beginning with the word `is` should return a boolean, not a list.

Reply:
Yes, it was a leftover from the first implementation. I'll change it accordingly.